test/fixtures/lightscript/comprehension/object-basic/actual.js

```diff
@@ -0,0 +1 @@
+{for i=0; i<10; i++: (i,i)}
```

Comment:
It's your show, but I don't fully agree with this. I think a key philosophical point in eliminating the "big table of for" is that good old fashioned for loops are perfectly acceptable for just counting numbers.